xmake/plugins/project/vsxmake/vsproj/templates/vcxproj/XmakeConfig(mode,arch)

```diff
@@ -1,3 +1,6 @@
+  <PropertyGroup Label="Configuration" Condition="'$(Configuration)|$(Platform)'=='#mode#|#vsarch#'">
+    <PlatformToolset>#toolset_version#</PlatformToolset>
+  </PropertyGroup>
```

Comment:
we need not it too. Xmake.props has added it.

https://github.com/xmake-io/xmake/blob/a70d5148d402195a411c590fd049c045ed5b5c7b/xmake/plugins/project/vsxmake/vsproj/Xmake.props#L54-L61

Reply:
`<PlatformToolset Condition="'$(VisualStudioVersion)' == '17.0'">v143</PlatformToolset>` has been added on master/dev. 

see https://github.com/xmake-io/xmake/issues/1838